winforms/src/toga_winforms/widgets/webview.py

```diff
@@ -75,14 +75,16 @@ def winforms_initialization_completed(self, sender, args):
             settings = self.native.CoreWebView2.Settings
             self.default_user_agent = settings.UserAgent
 
-            debug = True
+            debug = False
+            settings.AreBrowserAcceleratorKeysEnabled = debug
```

Comment:
This disables certain keyboard shortcuts:
- Ctrl-F and F3 for Find on Page
- Ctrl-P for Print
- Ctrl-R and F5 for Reload
- Ctrl-Plus and Ctrl-Minus for zooming
- Ctrl-Shift-C and F12 for DevTools
- Special keys for browser functions, such as Back, Forward, and Search

This aligns with pywebview's behavior.